monero-outreach-docs/translations/pt-br/storyboard/tool_for_renewal_pt_br.md

```diff
@@ -0,0 +1,77 @@
+
+**Aceitando Monero (para negócios).** A comunidade global de desenvolvedores por trás do Monero tem criado ferramentas para capacitar empresas a começarem a aceitar Monero por produtos e serviços. Cortar os altos custos de intermediários e deixar seus clientes pagar frações de centavos em taxas. Aprenda mais:
+
+- [Monero Outreach Merchant Services](https://www.monerooutreach.org/merchants/)
+- [https://web.getmonero.org/community/merchants/](https://web.getmonero.org/community/merchants/)
+- [https://www.projectcoralreef.com/](https://www.projectcoralreef.com/)
+
+### _Usando Monero_
+
+**Melhores Práticas.** Ao contrário do Bitcoin, Monero é designado para ser completamente privado e irrastreável. São habilidades básicas que deve ser aprendidas para tirar o máximo de proveito do Monero. [Aprenda mais sobre as melhores práticas do Monero.](https://www.monerooutreach.org/monero_best_practices/)
+
+**Enviando e pagando com XMR.** Esta é uma função básica de qualquer carteira Monero. Você precisa do endereço (ou um QR code) do destinatário a quem você deseja enviar fundos e precisa ter saldo em Monero na sua carteira. Se você estiver tendo problemas, há dezenas de tutoriais e forúns online com boas pessoas que podem ajuda-lo com isto.
+
+**Usando Monero em vez de Bitcoin ou moeda fiduciária.** [XMR.to](https://xmr.to/) permite a qualquer um (nas regiões suportadas) enviar/pagar em Bitcoin com seu Monero. Eles tem provado serem fidedignos e tem uma reputação de ótimo atendimento ao cliente. Você pode também usar XMR.to para comprar cartões de presente de qualquer fornecedor ou serviço que permite você comprar cartões de presente com Bitcoin, dando a você maior poder de compra nesses estabelecimentos – poder de compra que começa com Monero.
+
+### _Usando Monero para Mudar o Mundo_
+
+Se você chegou até aqui, você provavelmente sente como fazemos com que o Monero tenha o poder para mudar o mundo para melhor. É importante entender que Monero é liderado por voluntários, uma comunidade de código aberto. Se você se sentir atraído em seguir em frente contribuindo com seu tempo, talento, ou riqueza para salvaguardar seu futuro, [aprenda como você pode iniciar ajudando Monero](https://www.monerooutreach.org/stories/getting-started-helping-monero.html).
+
+Em um mundo de incertezas, nós temos a responsabilidade de lutar por ferramentas que nos deixe viver livres de opressão. A única maneira de encontrar a paz será quando tivermos poder para nos livrar das garras de governos opressores e bancos centrais e nos unirmos com pessoas que desejam viver livremente. Até sermos responsáveis por nosso próprio futuro e resgatar nossa independência social e financeira, não haverá justiça e não haverá paz.
+
+Conheça a justiça. Conheça a paz. Saiba que Monero irá salvaguardar nosso direito humano a privacidade e liberdade.
```

Comment:
Maybe swtich "salvaguardar" to "preservar"? Just another option, but it's up to you ;)

Reply:
Yes, sure!